plugins/outputs/newrelic/newrelic.go

```diff
@@ -152,7 +158,29 @@ func init() {
 	outputs.Add("newrelic", func() telegraf.Output {
 		return &NewRelic{
 			Timeout: internal.Duration{Duration: time.Second * 15},
-			Client:  http.Client{},
+			// TODO Don't set the client here, wait
+			// client: http.Client{},
```

Comment:
Please get rid of the TODOs! I don't want to get half-finished stuff in and let the TODOs pile-up in the repo. It is also not clear what needs to be done for this TODO to vanish. Waiting probably won't solve the problem... ;-)

Reply:
Done